test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageControllerSpec.scala

```diff
@@ -1739,8 +1739,29 @@ class SingleDisposalsTriageControllerSpec
         }
 
         val result = performAction()
-        status(result)          shouldBe OK
-        contentAsString(result) shouldBe "Indirect disposals not handled yet"
+        status(result) shouldBe SEE_OTHER
```

Comment:
we have a handy `checkIsRedirect` method you can use here which takes a `Future[Result]` and a `Call`